app/types/framework.tsx

```diff
@@ -18,26 +18,14 @@ type WidgetType = 'excerptWidget'
     | 'multiselectWidget'
```

Comment:
We should try to move the types used on new system to newAnalyticalFramework.tsx

Reply:
@tnagorra this is not used